### PR TITLE
ignore verify the server's TLS certificate

### PR DESCRIPTION
**Pull request by KerollesFathy**

### CODE REVIEW

File: ebda_integration/ebda_integration/ebda_api.py [hunk 1]
```diff
@@ -35,12 +38,12 @@ def auth(self):
     def check_session(self):
         "params: token | return (status_code, message)"
         url = self.base_url + self.check_session_url
-        params = frappe._dict({"token": self.settings.get_password("token", False)})
+        json = frappe._dict({"token": self.settings.get_password("token", False)})
```
Comment (meaziz) [inline ebda_integration/ebda_integration/ebda_api.py]: naming it json **is not** a good idea 👀

File: ebda_integration/ebda_integration/ebda_api.py [hunk 2]
```diff
@@ -35,12 +38,12 @@ def auth(self):
...
         try:
-            response = requests.get(url, params=params)
+            response = requests.post(url, json=json, verify=False)
             response.raise_for_status()  # Raises HTTPError for non-200 status codes
             response = response.json()
-            if response.status_code == 200 and response.message == "Token is active":
+            if response.get("status_code") == 200 and response.get("message") == "Token is active":
```
Comment (meaziz) [inline ebda_integration/ebda_integration/ebda_api.py:46]: Hypothetically according to the standards we should check on the status_code.. the message in this case may not match. if they changed the API.

File: ebda_integration/ebda_integration/utils.py [hunk 3]
```diff
@@ -64,7 +65,7 @@ def get_answers_from(survays: list = []) -> list:
 def update_answer_key_value(dictionary):
```
Comment (meaziz) [inline ebda_integration/ebda_integration/utils.py]: "dictionary" 👀 is a data type ..

**Review by meaziz — CHANGES_REQUESTED** [review]

File: ebda_integration/ebda_integration/ebda_api.py (hunk 1 above)
Comment (KerollesFathy) [reply ebda_integration/ebda_integration/ebda_api.py]: @meaziz u are right

File: ebda_integration/ebda_integration/ebda_api.py (hunk 2 above)
Comment (KerollesFathy) [reply ebda_integration/ebda_integration/ebda_api.py:46]: @meaziz the response at documentaion like this:
`{
      "status_code": 200, 
      "message": "Token is Active"
}`
how we can check this safely ?

File: ebda_integration/ebda_integration/utils.py (hunk 3 above)
Comment (KerollesFathy) [reply ebda_integration/ebda_integration/utils.py]: @meaziz yes :laughing: i will improve it using `update_answer_key_value(new_param_name : dict)`

**Pull request merged**